### PR TITLE
Add recipe for unite cms bundles

### PR DESCRIPTION
**Pull request by franzwilding**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

**Comment by franzwilding** [comment]
bump

File: unite-cms/core-bundle/0.10/manifest.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "bundles": {
+        "Symfony\\Bundle\\TwigBundle\\TwigBundle": ["all"],
+        "UniteCMS\\CoreBundle\\UniteCMSCoreBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
```
Comment (Nyholm) [inline unite-cms/core-bundle/0.10/manifest.json]: What about the assets? And package.json and webpack?

File: unite-cms/core-bundle/0.10/post-install.txt [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<bg=blue;fg=white>              </>
+<bg=blue;fg=white> What's next? </>
+<bg=blue;fg=white>              </>
+
+  * Create a database*
+    1. We enabled doctrine ORM adapter as a default. You can change this in config
```
Comment (Nyholm) [inline unite-cms/core-bundle/0.10/post-install.txt]: Is this done in the bundle? I dont see it on the recipe.

**Comment by Nyholm** [comment]
Friendly ping

**Comment by Nyholm** [comment]
Closing due inactivity. Feel free top open a new PR if you want to continue the work.

**Pull request closed without merging**